### PR TITLE
can now require('pantheon-helpers') through couch or node

### PR DESCRIPTION
**Pull request by dgreisen-cfpb**
requiring through couch and through node returns the same
export.

updated templates accordingly

### CODE REVIEW

File: templates/design_doc/src/lib/actions.coffee [hunk 1]
```diff
@@ -1,5 +1,5 @@
-do_action = require('./shared/do_action')
-validate_doc_update = require('./shared/validate_doc_update').validate_doc_update
+do_action = require('./pantheon-helpers').design_docs.do_action
+validate_doc_update = require('pantheon-helpers').design_docs.validate_doc_update
```
Comment (m3brown) [inline templates/design_doc/src/lib/actions.coffee]: Are you sure this works? To me it looks like [design_docs.validate_doc_update](https://github.com/dgreisen-cfpb/pantheon-helpers/blob/master/lib/pantheon-helpers-design-docs/validate_doc_update.js) is an object that contains the function `validate_doc_update`, meaning this should be:

```
validate_doc_update = require('pantheon-helpers').design_docs.validate_doc_update.validate_doc_update
```

**Comment by m3brown** [comment]
I don't think this is necessarily a change in this PR, but my attention was brought to it when looking at the kanso.json file:  Is it not possible to specify the version of external libs (i.e. underscore) used in design docs and/or sync to the package.json version?
1. The version is not specified anywhere (e.g. kanso.json)
2. The version of libs in node_modules and in design_docs could have a delta that would not be obvious or trackable

I'm not sure whether or not these concerns are worth fretting over, it just seems kind of haphazard right now.

File: templates/design_doc/src/lib/actions.coffee (hunk 1 above)
Comment (dgreisen-cfpb) [inline templates/design_doc/src/lib/actions.coffee]: good catch. fixed.

**Comment by dgreisen-cfpb** [comment]
I would love to freeze versions. however, I don't think I can get it to work, since not actually using the package manager for any but two modules.

plus, since kanso is dead, we don't have to worry about any updates :D

**Pull request merged**